src/tests/tracing/eventpipe/processinfo2/processinfo2.csproj

```diff
@@ -11,9 +11,11 @@
          Issue: https://github.com/dotnet/runtime/issues/39935
     -->
     <IlasmRoundTripIncompatible>true</IlasmRoundTripIncompatible>
+    <EventSourceSupport Condition="'$(TestBuildMode)' == 'nativeaot'">true</EventSourceSupport>
```

Comment:
We keep adding these - might be time to add a Directory.Build.props to src/tests/tracing (or src/tests/tracing/eventpipe?) that has this line instead.